starter_code/index.html

```diff
@@ -11,37 +11,42 @@
 		<link rel="stylesheet" href="css/style.css">
 
 	</head>
-	<header id="header">
+	<header>
 		<h1 id="logo">Relaxr</h1>
-		<h3 id="headerMessage">Get piece of mind with a single tap</h3>
-		<button id="button"><a href="TBA">Get it Now</a></button>
+		<div id="top">
+			<h3 id="headerMessage">Get piece of mind with a single tap</h3>
+			<a rel="" id="getNow">Get it Now</a> 
+		</div>
 	</header>
 
 	<body>
 		<div id="benifits">
-			<p>The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
+			<h3>Benifits</h3>
+			<p id="bmessage">The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
 			<ul id="list">
 				<li>Schedule meetings for you</li>
 				<li>Excel automation to complete your work for you</li>
 				<li>Responds to emails on your behalf</li>
 				<li>Does all your work for you with our revolutionary Ai technology</li>
 			</ul>
 		</div>
-		<div id="review">
-			<p>"Relaxr changed my life. I've been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work."</p>
-			<p>- Amanda, Intuit</p>
+		<div id="testimonial">
+			<div id="testemony">
+				<p>"Relaxr changed my life. I've been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work."</p>
+				<p id="testemonyName">- Amanda, Intuit</p>
+			</div>
 		</div>
 		<div id="signUp">
-			<button id="signUp">Sign Up Now!</button>
+			<a href="" id="signUpLink">Sign Up Now!</a>
 		</div>
 	</body>
 
-	<footer id="footer">
-		<h3>Relaxr</h3>
+	<footer>
+		<h3 id="footerName">Relaxr</h3>
 		<nav id="socialMedia">
-			<a href="https:/twitter.com">twitter logo</a>
-			<a href="https://facebook.com">facebook logo</a>
+			<a href="https:/twitter.com"><img src="/images/twitter.png" alt="twitter"></a>
```

Comment:
no need for "/" before images because you're going to access the images folder directly